packages/stream/create-stream/create-stream.test.js

```diff
@@ -0,0 +1,238 @@
+import nock from 'nock'
+import { expect } from 'chai'
+import { createStream } from './create-stream.js'
+import { Internal } from '../../internal/index.js'
+import { init } from '../../app/init/init.js'
+
+describe('Create Stream Module', function () {
+  describe('Basic test', function () {
+    it('should be a function', function () {
+      expect(createStream).to.be.a('function')
+    })
+  })
+
+  describe('Negative test', function () {
+    afterEach(function () {
+      return nock.cleanAll()
+    })
+
+    it('should return error if initialization instance & config not input', async function () {
+      try {
+        await createStream()
+      } catch (error) {
```

Comment:
Same with the [comment on this PR](https://github.com/inlivedev/inlive-js-sdk/pull/12/files#r973824303). We shouldn't use `try catch` for testing